changelogs/fragments/add_template_path.yaml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - Allows the cli_parse module to find parser.template_path inside roles or collections when a path relative to the role/collection directory is provided.
```

Comment:
Collections do not support a top-level "templates" directory, I believe. Are you sure this works?